NodeFirefox/Dockerfile.txt

```diff
@@ -38,3 +38,7 @@ RUN sudo echo ""
 
 # Generating a default config during build time
 RUN /opt/bin/generate_config > /opt/selenium/config.json
+
```

Comment:
I wonder if it is better to install the package when the user is root and avoid using sudo... If you install it when the user is root, are the results the same @pabloFuente? 
If so, I think it would be better to install it with root.